docs/topics/api.rst

```diff
@@ -134,3 +134,32 @@ for the client.
 
 .. _`token generator`: https://solitude.readthedocs.org/en/latest/topics/braintree.html#generate-a-token
 .. _`Solitude`: https://github.com/mozilla/solitude/
+
+Subscribe
+~~~~~~~~~
+
+This endpoint allows you to subscribe a buyer to a Braintree plan.
+To pay using a new credit card, submit ``pay_method_nonce``. To pay
+with a saved credit card, submit ``pay_method_uri``.
+
+.. http:post:: /api/braintree/subscribe/
+
+    **Request**
+
+    :param string pay_method_nonce:
+        A single use token representing the buyer's submitted payment method.
+        In the buy flow, the Braintree JS client intercepts a form
+        submission (e.g. credit card numbers), obfuscates it, and gives us
+        this string.
+    :param string pay_method_uri:
+        Solitude URI to an existing `payment method`_ for this buyer.
+        When paying with a saved card, post this value instead of a nonce.
```

Comment:
Does sending pay_method_uri and pay_method_nonce at the same time raise an error?